.github/workflows/build_deploy.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - name: Get yarn cache directory path
         id: yarn-cache-dir-path
         working-directory: submodules/volto
-        run: echo "::set-output name=dir::$(yarn cache dir)"
+        run: echo "{name}={value}" >> $GITHUB_OUTPUT
```

Comment:
@stevepiercy I think this should be changed to:

run: echo "{dir}={$(yarn cache dir)}" >> $GITHUB_OUTPUT

@sneridagh @ericof    That's quite a few workflows we maybe have to check

Reply:
D'oh! That's what happens when we go on auto-pilot and copy-pasta. Fixed in [8e0614a](https://github.com/plone/documentation/pull/1439/commits/8e0614a6773a15858e6adc54c279bc0495df7f6e). Please check again.